src/libraries/MathLib.sol

```diff
@@ -10,11 +10,13 @@ int256 constant WAD_INT = int256(WAD);
 /// @custom:contact security@morpho.org
 /// @notice Library to manage fixed-point arithmetic on signed integers.
 library MathLib {
-    function wMulDown(int256 a, int256 b) internal pure returns (int256) {
+    /// @dev Returns the multiplication of `a` by `b` (in WAD) rounded towards 0.
+    function wMulTo0(int256 a, int256 b) internal pure returns (int256) {
```

Comment:
My two cents/nitpick: using the full word "zero" would maybe be more clean.

1) it's easy to get confused between char `0` and `o` and `O`. Having "zero" directly in the name is also much more explicit
2) probably it's not comfortable to have a number inside a function when you need to type it

Note: The solidity style guide does not have a strong opinion about having numbers inside the function's name